tests/lib/qunit/addons/composite/qunit-composite.js

```diff
@@ -39,6 +39,10 @@ function runSuite( suite ) {
 	}
 
 	QUnit.asyncTest( suite, function() {
+		// See: https://github.com/JamesMGreene/qunit-composite/issues/20
```

Comment:
Is this change required? Seems improper to me to make change directly in a lib file.